app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -39,6 +39,7 @@ ListingController = (
   $scope.lotterySearchNumber = ''
   $scope.smallDisplayClass = "small-display-none"
   $scope.lotteryRankingSubmitted = false
+  $scope.loadingLotteryResults = false
```

Comment:
that's a good point, and I'd agree... I don't think we're using different disabled/loading variables in very many places so it should be an easy refactor. but let's capture that separately, I've found that some bugs that they're finding in QA have come from some refactoring that caused some unexpected glitches that then need to get fixed (I'm thinking of one of the things I did w/ Account userAuth clearing messages)